lib/doofinder_api_landing.php

```diff
@@ -32,11 +32,11 @@ public function __construct($hashid, $api_key, $region)
     /**
      * Make a request to the API to get landing data
      *
-     * @param array Product data
+     * @param string Name from Landing
      */
     public function getLanding($slug)
     {
-        $endpoint = '/plugins/landing_new/' . $this->hashid . '/' . $slug;
+        $endpoint = "/landing/{$this->hashid}/{$slug}";
```

Comment:
I'm not 100% sure if the brackets are required in the last one, but it if works, it's okay :smile: 

```suggestion
        $endpoint = "/landing/{$this->hashid}/$slug";
```

Reply:
Tienes razón, funciona pero no es necesario. Lo cambio!